addons/mail/models/mail_thread.py

```diff
@@ -2236,6 +2236,26 @@ def _message_create(self, values_list):
             self = self.with_context(ctx)
         return self.env['mail.message'].create(create_values_list)
 
+    def _message_get_default_subject(self, record_name=None):
+        """ Helper to get the default subject when creating a message on this thread
+            For most records it should just be the name of the record,
+            override in child model when it does not make sense
+            :example:
+                'About survey: what did you think?'
+            :param record_name: string of the name of the record, replaced by current name
+            :return: the subject of the message as a string
+        """
+        record_name = record_name or ''
```

Comment:
I would make this method a true multi, receiving record_names which could be a dict (record.id -> name). That way we could do a name_get in batch for example, when moving all methods to true multi. Currently most methods are singleton methods but we plan to move towards a more batch-enabled code soon :) .